src/AutoPilotPlugins/APM/APMCameraComponent.cc

```diff
@@ -62,5 +62,5 @@ QUrl APMCameraComponent::setupSource(void) const
 
 QUrl APMCameraComponent::summaryQmlSource(void) const
 {
-    return QUrl::fromUserInput(QStringLiteral("qrc:/qml/APMCameraComponentSummary.qml"));
```

Comment:
How come this is removed?

Reply:
A lot of parameters are outdated there, so it isn't functional right now for the most of it for Ardupilot, so I thought it would be better disabled and force the users to do it by parameters. As part of https://github.com/mavlink/qgroundcontrol/pull/10667 we will go over that menu as well, and leave it updated for the current Ardupilot setup. Is that fine?

Reply:
Sure